WalletWasabi.Fluent/Models/UI/ApplicationSettings.cs

```diff
@@ -55,7 +54,6 @@ public partial class ApplicationSettings : ReactiveObject
 
 	// General
 	[AutoNotify] private bool _darkModeEnabled;
-
 	[AutoNotify] private bool _autoCopy;
```

Comment:
This one also should be renamed to fix the error in `NullApplicationSettings.cs`